Stack/Opc.Ua.Core/Security/Certificates/CertificateFactory.cs

```diff
@@ -112,12 +112,17 @@ public class CertificateFactory
 {
     #region Public Constants
     /// <summary>
-    /// The default certificate factory security parameter.
+    /// The default certificate factory security parameters.
     /// </summary>
-    public const ushort defaultKeySize = 2048;
-    public const ushort defaultHashSize = 256;
-    public const ushort defaultLifeTime = 12;
+    /// <remarks>
+    /// The security parameters may change over time,
+    /// so they are only readonly values, not constants.
+    /// </remarks>
+    public static readonly ushort DefaultKeySize = 2048;
+    public static readonly ushort DefaultHashSize = 256;
+    public static readonly ushort DefaultLifeTime = 12;
     #endregion
```

Comment:
ok, this is a breaking change, but the naming convention was not quite right and the Defaults should be readonly not constants, so using the opportunity to fix this...